.github/workflows/ci-pr-only-cov.yml

```diff
@@ -1,21 +1,18 @@
-name: CI PR Only COV
+# Copied & adapted (job name, version label) from
+# https://github.com/ytanikin/PRConventionalCommits?tab=readme-ov-file#usage-with-labeling-where-label-is-just-a-task-type
+
+name: PR Conventional Commit Validation
 
 on:
   pull_request:
-    types:
-      - ready_for_review
-      - edited
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref && github.ref || github.run_id }}
-  cancel-in-progress: true
+    types: [opened, synchronize, reopened, edited]
 
 jobs:
   validate-pr-title:
     name: Validate PR Title
     runs-on: ubuntu-latest
     steps:
       - name: PR Conventional Commit Validation
-        uses:  ytanikin/PRConventionalCommits@8d258b54939f6769fcd935a52b96d6b0383a00c5
```

Comment:
You need to check with @slawomirbabicz if this is fine.

Reply:
@marko-k0 do you mean for security reasons? it's the same commit, does it still need to be checked?

Reply:
Yes. Action was reviewed by @slawomirbabicz and this revision was given.